sandstone/app.py

```diff
@@ -14,13 +14,14 @@
 from sandstone import settings
 from sandstone.lib import ui_methods
 from sandstone.lib.app_loader import get_installed_app_static_specs
+from sandstone.lib.handlers.broadcast import BroadcastManager
 import sandstone.urls
 from sandstone.urls import URL_SCHEMA
-
+from filewatcher import Filewatcher
 
 
 class SandstoneApplication(tornado.web.Application):
-
+    filewatcher_instance = Filewatcher()
```

Comment:
The `FileWatcher` instance should be instantiated within the `filebrowser` app, and not within the core framework.